docs/development.md

```diff
@@ -181,6 +181,33 @@ zk test db
   **Note**. If you have compilation issues with `sqlx`, then make sure to run `zk up` before running the tests. Also, if
   you see some tests fail, you might need to call `zk db reset` and restart the tests.
 
+### Code coverage
+
+**Note**. Code coverage measurement requires `grcov` and `llvm-tools-preview` to be installed.
+
+```bash
+cargo install grcov
+rustup component add llvm-tools-preview
+```
+
+To measure code coverage of unit tests, just set the environment variable `CODE_COVERAGE` to `true`.
```

Comment:
not true strictly speaking as setting it to true should result in `'true'` not `true` and thus `CODE_COVERAGE=false zk test prover` will also work (as `'false'` is truthy) afaik.

Reply:
Absolutely true, thank you for highlighting it

Reply:
Done